actions.yaml

```diff
@@ -47,3 +47,13 @@ create-backup:
 list-backups:
   description: |
     Lists backups in s3 storage.
+restore-backup:
+  description:  |
+    Restore a Synapse backup.
+    S3 credentials are retrieved from a relation with the S3 integrator charm.
+    The server_name and filesystem configuration for the application should be
+    identical to the application restored.
+  params:
+    backup-id:
+      type: string
+      description: A backup-id to identify the backup to restore.
```

Comment:
s/A backup-id to identify/The backup-id to identifying

Reply:
Changed s/A/The as to identify is correct. Thanks!